### ISSUE TITLE
Explain properties of a menu entry in more detail

### ISSUE THREAD

**Issue by digitalcraftsman** [rdwatters/hugo-docs-concept#109]
[Listing the properties and it's type of a menu entry](https://hugodocs.info/content-management/menus/#what-is-a-menu-in-hugo) isn't the best description even if most of the properties are obvious. Explicit is better an implicit.

**Comment by rdwatters** [comment]
Agreed. I'll work on this over the weekend. I also need to get on the new Quick Start sooner rather than later.
